surveyor.gemspec

```diff
@@ -31,7 +31,7 @@ Gem::Specification.new do |s|
   s.add_development_dependency('poltergeist', '~>1.17.0')
   s.add_development_dependency('json_spec', '~> 1.1.1')
   s.add_development_dependency('factory_bot', '~> 4.8.0')
-  s.add_development_dependency('database_cleaner', '~> 1.2.0')
+  s.add_development_dependency('database_cleaner', '~> 1.2')
```

Comment:
I think a bunch of these seem to not match the rails app

Reply:
Probably -- but we don't use this gem in our app (this is only a dev dependency) & was causing a warning.